### PR TITLE
Adding unmarshal method

### PR DESCRIPTION
**Pull request by tamccall**
trying to merge code piece wise from the update ticket field pr

### CODE REVIEW

File: zendesk/resource_zendesk_ticket_field.go [hunk 1]
```diff
@@ -168,6 +168,124 @@ func resourceZendeskTicketField() *schema.Resource {
 	}
 }
 
+func marshalTicketField(d identifiableGetterSetter) (client.TicketField, error) {
```
Comment (nukosuke) [inline zendesk/resource_zendesk_ticket_field.go]: Is this unmarshal?
Sorry, I'm a little bit confused.

- `TicketField` => `ResourceData`
- `ResourceData` => `TicketField`

Which is marshal?

Comment (tamccall) [reply zendesk/resource_zendesk_ticket_field.go]: yeah probably should change the name here. this is going resource data => ticketfield

**Comment by nukosuke** [comment]
Looks fine about the conversion method.

File: zendesk/resource_zendesk_ticket_field.go (hunk 1 above)
Comment (nukosuke) [reply zendesk/resource_zendesk_ticket_field.go]: OK, thanks.

**Pull request merged**